src/AppBundle/Twig/WikiExtension.php

```diff
@@ -69,6 +70,24 @@ public function userLink($username, $projectUrl, $label = null)
     }
 
     /**
+     * Get a link to the given user's userpage, or to Special:Contribs if $username is an IP
+     * @param  string $username   Username
+     * @param  string $projectUrl Project domain and protocol such as https://en.wikipedia.org
+     * @param  string [$label]    The link text, defaults to $username
+     * @return string Markup
+     */
+    public function groupLink($group, $projectUrl, $label = null)
+    {
+        if (!$label) {
+            $label = $group;
+        }
+        // Ignoring this inspection, as we want all of the output on one line.
+        // @codingStandardsIgnoreStart
+        return "<a href='$projectUrl/wiki/index.php?title=Special:ListUsers&group=$group&creationSort=1&limit=50' target='_blank'>$label</a>";
```

Comment:
I may have gotten rid of `linkWiki`, or at least I can't find it now. This was in favour of WikiExtension.php which has helpers for all sorts of links. In there we could add a path helper that fetches and caches the `articlepath` and `scriptpath` from [https://en.wikipedia.org/w/api.php?action=query&meta=siteinfo](meta=siteinfo). I will volunteer to do this since I apparently broke the old system =P